docs/_snippets/cohort-with-details-response-json-with-message.mdx

```diff
@@ -0,0 +1,35 @@
+```json
+{
+  "message": "Patient(s) successfully added to cohort",
+  "cohort": {
+    "id": "00000000-0000-0000-0000-000000000000",
+    "eTag": "1",
+    "name": "High Risk",
+    "description": "Patients that need frequent and robust monitoring.",
+    "settings": {
+      "monitoring": {
+        "hie": {
+          "enabled": false,
+          "frequency": "monthly"
+        },
+        "adt": true,
+        "pharmacy": {
+          "notifications": true,
+          "schedule": {
+            "enabled": false,
+            "frequency": "monthly"
```

Comment:
what's the default value when this was never set? shouldn't this be omitted? Fine if we want to indicate different possibilities (like the cx set it to monthly and enabled it, then disabled and enabled notification, so the frequency is still there), but that's the exception, so it should be the exception on the docs too.

ditto below and other files.